source/guides/branding.rst

```diff
@@ -64,7 +64,7 @@ General Mattermost Description
 
 When developing a Mattermost integration, the following description can be used to describe Mattermost:
 
-  Mattermost is an open source Slack-alternative built for enterprise. Thousands of companies use Mattermost for secure, compliant workplace messaging across web, PC and phones with archiving, search and integration with hundreds of third party applications. Available under MIT license in 16 languages, Mattermost offers peace-of-mind, value, control, and freedom from lock-in for organizations around the world. Learn more at https://mattermost.com/
+  Mattermost is an open source Slack-alternative built for enterprise. Thousands of companies use Mattermost for secure, compliant workplace messaging across web, PC and phones with archiving, search, corporate SSO, and integration with hundreds of third-party applications. Available under MIT license in 16 languages, Mattermost offers peace-of-mind, value, control, and freedom from lock-in for organizations around the world. Learn more at https://mattermost.com/.
```

Comment:
```suggestion
Mattermost is an open source Slack-alternative built for enterprise. Thousands of companies use Mattermost for secure, compliant workplace messaging across web, PC, and phones; with archiving, search, corporate SSO, and integration with hundreds of third-party applications. Available under MIT license in 16 languages, Mattermost offers peace-of-mind, value, control, and freedom from lock-in for organizations around the world. Learn more at https://mattermost.com/.
```

Reply:
"Thousands of companies use Mattermost for secure, compliant workplace messaging across web, PC, and phones; with archiving, search, corporate SSO, and integration with hundreds of third-party applications." is a little unclear. Do companies use Mattermost across web, PC, and phones with the functionality to support archiving, search etc or do they use it across web, PC, and phones and then archiving, search etc are the reasons for using Mattermost on those devices/in general? 

Also wondering whether "data-sovereignty", "compliance", and "reporting" should be in there somewhere? 

3/5 on improving clarity via punctuation. 0/5 on rewording it because that's a bigger discussion beyond this PR.

Reply:
Correction, I totally glossed over "compliant" in the first sentence. But the other two suggestions still stand. :)

Reply:
I believe the intent is the former - companies use Mattermost across devices, and Mattermost supports features like archiving, search, ...

I like the "data-sovereignty" and "reporting" suggestions, but agree that would be a broader discussion. :)

Reply:
Great. I updated my suggestion slightly.

Reply:
Given this is a text that we recommend people to use as a short description for work they've done, e.g. an integration, wondering if "leveraging" is easily understood by non-English audience?

Otherwise the change looks good to me :)